activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -518,24 +520,25 @@ def drop_table(table_name, options = {})
       # agnostic and should usually be avoided.
       #
       # Available options are (none of these exists by default):
+      # * <tt>:comment</tt> -
+      #   Specifies the comment for the column. This option is ignored by some backends.
+      # * <tt>:collation</tt> -
```

Comment:
I suppose the options were listed by roughly common (limit, default, null, etc) to uncommon (collation, comment).
Is there any benefit to change the order? (at least I'd prefer the order as before)

Reply:
This is the alphabetical order